packages/styling/src/styles/theme.ts

```diff
@@ -107,16 +107,17 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
     ...theme.semanticColors
   };
 
-  let defaultFontStyles: IFontStyles = DefaultFontStyles;
+  let defaultFontStyles: IFontStyles = { ...DefaultFontStyles };
+
   if (theme.defaultFontStyle) {
-    for (const fontStyle of Object.keys(DefaultFontStyles)) {
-      defaultFontStyles[fontStyle] = { ...defaultFontStyles[fontStyle], ...theme.defaultFontStyle };
+    for (const fontStyle of Object.keys(defaultFontStyles)) {
```

Comment:
Couldn't you iterate over `DefaultFontStyles` instead and initialize `defaultFontStyles` to `{}`?